README.md

```diff
@@ -33,6 +33,13 @@ There are optional environment variables for other build systems as well:
 * COVERALLS_RUN_AT  (a date string for the time that the job ran.  RFC 3339 dates work.  This defaults to your
 build system's date/time if you don't set it.)
 * COVERALLS_PARALLEL (more info here: https://coveralls.zendesk.com/hc/en-us/articles/203484329)
+### [Jest](https://facebook.github.io/jest/)
+- Install [jest](https://facebook.github.io/jest/docs/en/getting-started.html)
+- Use the following to run tests and push files to coveralls: 
+```
+jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
```

Comment:
Might want to just add `coverage/` to the `.gitignore` instead of removing the whole directory.

Reply:
I'll have to review my use case for this but I don't think adding coverage to my gitignore does the same thing.

Reply:
OK. I figure Jest writes it there for some caching and optimization. It also allows you to check it again at a later time without re-running all the test. Small stuff, no worries.